src/ESP8266LittleFS.java

```diff
@@ -320,7 +321,17 @@ private void createAndUpload(){
     if(isNetwork){
       System.out.println("[LittleFS] IP      : "+serialPort);
       System.out.println();
-      sysExec(new String[]{pythonCmd, espota.getAbsolutePath(), "-i", serialPort, "-s", "-f", imagePath});
+
```

Comment:
Do you think you could actually see if an AUTH is needed before presenting the box?  I understand that you're using it, but since ArduinoOTA is so insecure anyway I have a feeling that most people will not be using a password at all.

It's possible to probe the board using UDP comms (connect to ESP8266, send SPIFFS command, check for AUTH response, see espota.py), or even just call  espota.py once, and if it fails then prompt for a PWD and try again.

Reply:
Very nice and simple, BTW!  Just trying to make the default case the easiest/simplest for end users.